kernel/solo5.h

```diff
@@ -136,8 +165,13 @@ uint64_t solo5_clock_monotonic(void);
 uint64_t solo5_clock_wall(void);
```

Comment:
... in other words `solo5_clock_xxx()` can never fail. If they do for some reason then that is legitimately an unrecoverable (and fatal) error.